salt/apps/ocw/sync_repo.sls

```diff
@@ -0,0 +1,31 @@
+configure_ocw_src_git_sparsecheckout:
+  git.config_set:
+    - name: core.sparsecheckout
+    - value: True
+    - cwd: /usr/local/Plone/zeocluster/src
+
+configure_ocw_publishing_git_sparsecheckout:
+  git.config_set:
+    - name: core.sparseCheckout
+    - value: True
+    - cwd: /usr/local/Plone/zeocluster/publishing
+
+add_ocw_publishing_to_sparsecheckout:
+  file.managed:
+    - name: /usr/local/Plone/zeocluster/src/.git/info/sparse-checkout
+    - contents: 'plone/publishing'
+
+add_ocw_src_to_sparsecheckout:
+  file.managed:
+    - name: /mnt/ocwfileshare/OCWEngines/.git/info/sparse-checkout
+    - contents: 'plone/src'
```

Comment:
These contents are swapped between the target directories.

Also, this block should probably be conditional based on whether the target directory exists.